modules/ROOT/pages/sync/binding.adoc

```diff
@@ -5,3 +5,7 @@ include::{partialsdir}/attributes.adoc[]
 = Binding a {mobile-client} with the {service-name} Service
 
 include::{partialsdir}/generic-binding.adoc[]
+
+== Next Steps
+
+* xref:sync/adding-sync-to-your-app.adoc[Adding {service-name} Service to your Application]
```

Comment:
`service-name` is undefined here, you need to either add `:service-name: Data Sync` to the top of the document or use `:sync-service:` which is defined in the main attributes.adoc.

Reply:
@pb82 I don't understand. I have `:service-name: Data Sync` at the top of the document in the exact same way as the other docs.

Reply:
@darahayes Strange, when i render it it shows up as this:

![bildschirmfoto 2018-07-10 um 12 02 54](https://user-images.githubusercontent.com/1851198/42503557-45d56b86-8439-11e8-88c6-f870215dd7fa.png)

But service-name is defined

Reply:
It looks like using an include after defining a variable makes the variable undefined after the include. To fix this you can use `Adding {sync-service} Service` instead.